src/molecules/Modal/ModalHeader.native.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+import Space from '../../atoms/Space';
+import Size from '../../atoms/Size';
+import Flex from '../../atoms/Flex';
+import View from '../../atoms/View';
+import Button from '../../atoms/Button';
+
+const FullScreenModalHeader = styled(View)`
+  background-color: ${(props) => props.theme.colors.background[200]};
```

Comment:
why can't you use our card and override just the border-radius?

Reply:
I honestly feel that the card is not the correct component to use here. Further, the shadow constructs are not the same and nor can I override the card border.